src/openrct2/drawing/Text.cpp

```diff
@@ -229,6 +229,26 @@ int32_t gfx_draw_string_left_wrapped(
     return layout.GetHeight();
```

Comment:
Make the old function call the new one to avoid code duplication

Reply:
Please make the old function call the new one, like I said, to avoid code duplication

Reply:
Sorry missed this, will do in next commit!